src/main/java/edu/cornell/kfs/sys/CUKFSPropertyConstants.java

```diff
@@ -80,9 +80,16 @@ public class CUKFSPropertyConstants {
     public static final String FINALIZED_DATE = "finalizedDate";
 
     //** CU Generic ISO-FIPS Country modification items **
-    public static class ISOFIPSCountryMap {
+    public static class Location {
```

Comment:
Asking: Would using "Nation" instead of "Location" make this more self documenting when the strings are used elsewhere in the code?
I could go either way on this. Asking the question since my mind jumps to either asset location placement in buildings or location services at a street level and not the higher level concept of a country when I see that term.